src/main/resources/wa-task-cancellation-employment-et_englandwales.dmn

```diff
@@ -46,13 +46,13 @@
       </rule>
       <rule id="DecisionRule_0s533mm">
```

Comment:
Hi Kala,
Would it make sense to delete the extra rows that are now identical to previous rows?

Reply:
Hi Fergus, 

Deleted identical rows